additionalCSS/workSection.css

```diff
@@ -0,0 +1,123 @@
+.workTextContainer {
+  display: flex;
+  justify-content: center;
+}
+
+.workText {
+  font-family: "Inter", sans-serif;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  padding-top: 22px;
+  width: 70%;
+}
+
+.indicator {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+}
+
+.indicatorImg {
+  width: 48px;
+  height: 4px;
+  border-radius: 24px;
+  background: #36b37e;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.workContainer {
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: repeat(5, 474px);
+  row-gap: 24px;
+  justify-content: center;
+  padding-top: 62px;
+}
+
+.firstContainer {
+  border-radius: 8px;
+  background: #ebf0ee;
+  outline: 1px solid #d0d9d4;
+}
+
+.secondContainer {
+  width: 327px;
+  height: 222px;
+  border-radius: 8px;
+  margin-top: 252px;
+  background: #fff;
+  padding-top: 16px;
+}
```

Comment:
- Kindly Make sure to use Flexbox for each card of the work section according to [this requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.)

Reply:
@GedeonTS I would appreciate your clarification on this one too.👍🏿